src/components/editor/StartingPoints.jsx

```diff
@@ -40,14 +30,27 @@ class StartingPoints extends Component {
   onDropFile(files) {
     // supplies the json loaded from the resource template
     const handleFileRead = () => {
-      const content = fileReader.result
-      const valid = this.validateTemplate(JSON.parse(content))
-      if (valid) {
-        this.props.setResourceTemplateCallback(content)
-      }
-      else {
-        // TODO: don't continue on with display of ResourceTemplate component (#295)
-        console.error("ERROR: unable to use template as it isn't valid - we shouldn't display form")
+      let template
+      try {
+        template = JSON.parse(fileReader.result)
+        var schemaUrl = template.schema || (template.Profile && template.Profile.schema)
+        if (schemaUrl == undefined) {
+          if (template.Profile) {
+            schemaUrl = "https://ld4p.github.io/sinopia/schemas/0.0.2/profile.json"
+          } else {
+            schemaUrl = "https://ld4p.github.io/sinopia/schemas/0.0.2/resource-template.json"
+          }
+          console.error(`No schema url found in template. Using ${schemaUrl}`)
```

Comment:
It could be.  What would you prefer?  This message could be in addition to others -- it's just about "hey, you don't have a schema url in your profile/template, so i'm using the one from LC".